back/app/services/permissions/permissions_service.rb

```diff
@@ -0,0 +1,402 @@
+# frozen_string_literal: true
+
+module Permissions
+  class PermissionsService
```

Comment:
Maybe this name is a bit too broad. Did you consider making it more specific, like `ParticipationPermissionsService`?

We could also create a folder `permissions/participation` and put these 4 files (this file, `idea`, `initiative`, `project` permissions services) there. These 4 files seem to be quite different from `permissions_update` and `registration_requirements` services.

Reply:
In general, really nice refactoring! Makes a lot of sense.